MetadataExtractor/Formats/Icc/IccReader.cs

```diff
@@ -119,28 +123,28 @@ public IccDirectory Extract(IndexedReader reader)
             return directory;
         }
 
-        private static void Set4ByteString(Directory directory, int tagType, IndexedReader reader)
+        private static void Set4ByteString(Directory directory, int tagType, ref BufferReader reader)
```

Comment:
Would `in` instead of `ref` work here (and below)?

Reply:
The _position needs to be mutated when using the sequential access methods, so this needs to remain ref.

Reply:
Ah yes, makes sense 👍